src/Transpiler.ts

```diff
@@ -1778,8 +1778,10 @@ export abstract class LuaTranspiler {
             result += this.indent + `${className}.${fieldName} = ${value}\n`;
         }
 
-        // Try to find constructor
-        const constructor = node.members.filter(ts.isConstructorDeclaration)[0];
+        // Find first constructor with body
+        const constructor = node.members.filter(n => {
+            return ts.isConstructorDeclaration(n) && n.body;
```

Comment:
Also no reason here to have a `{ return }` lambda if you can just do `m => ts.isConstructorDeclaration(m) && m.body`

Reply:
I did use that but the line was to long with the typecast. Thought if I have to split the lambda it would be more readable to add a proper body.

Reply:
I prefer breaking expressions at periods instead of in the middle of lambdas, something like
```ts
node.members
    .filter(m => ts.IsConstructorDeclaration(m) && m.body)[0]
    as ts.ConstructorDeclaration
```